test/dockertest.go

```diff
@@ -0,0 +1,77 @@
+package test
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+)
+
+func RunPostgresDocker(t *testing.T) *dockertest.Resource {
+	pool, err := dockertest.NewPool("")
+	require.NoErrorf(t, err, "failed to create dockertest pool")
+
+	resource, err := pool.RunWithOptions(
+		&dockertest.RunOptions{
+			Repository: "postgres",
+			Tag:        "13.0",
+			Env: []string{
+				"POSTGRES_USER=postgres",
+				"POSTGRES_PASSWORD=postgres",
+				"POSTGRES_DB=bubbly",
```

Comment:
I feel these must come from the environment variables: `Tag`, `POSTGRES_USER`, `POSTGRES_PASSWORD`, `POSTGRES_DB`.

This would give us:

✔️ More flexibility
✔️ Better upgradeability
✔️ Better security

If you are _completely_ against the idea of having to set a few environment variables before running tests, 

👺 firstly, this; and
❓  at least let's consider reading the values from environmental variables while providing fall-back default values, such as the ones currently hard-coded, in case the environment variables are not set.

Reply:
Could we just use the values from `bCtx` the Bubbly Context? As those come from environment variables...
What do you think?

Reply:
Yes, sounds good to me!